frontend/src/views/FarmInformation/FarmInformation.tsx

```diff
@@ -39,8 +51,22 @@ export default function FarmInformation() {
   const [rawAnimalNames, setRawAnimalNames] = useState<{ [id: string]: string }>({});
 
   // Props for region selections
-  const [regionOptions, setRegionOptions] = useState<{ value: number; label: string }[]>([]);
-  const [subregionOptions, setSubregionOptions] = useState<{ value: number; label: string }[]>([]);
+  const [regionOptions, setRegionOptions] = useState<Array<SelectOption>>([]);
+  const [subregionOptions, setSubregionOptions] = useState<Array<SelectOption>>([]);
+
+  // Initialize year list
+  const [yearOptions, setYearOptions] = useState<Array<SelectOption>>([]);
```

Comment:
There's no need to have this as a useState variable set by a useEffect. You could initialize this with a useMemo.

Reply:
Good point, I will make the change.